pkg/timeutils/utils_test.go

```diff
@@ -0,0 +1,24 @@
+package timeutils
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestGetTimestamp(t *testing.T) {
+	now := time.Now()
+	cases := []struct{ in, expected string }{
+		{"0", "-62167305600"},
+		{"2015-12-05", "1449273600"},
+		{"invalid", "invalid"},
+		{"1m", fmt.Sprintf("%d", now.Add(-1*time.Minute).Unix())},
```

Comment:
`2006-01-02T15:04:05.999999999Z+07:00` is not a valid format.
`2006-01-02T15:04:05.999999999+07:00` is. [[RFC3339]](https://tools.ietf.org/html/rfc3339#page-5)